### ISSUE TITLE
Erbele crashes when closing window

### ISSUE THREAD

**Issue by abentele** [abentele/Erbele#45]
Affected version: Erbele 3.9

Steps to reproduce:
* open Erbele
* close main window
* quit application

=> Erbele crashes

Maybe it has to do with conversion of class FRADocumentsMenuController to Swift.

```
Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   org.erbele.Erbele             	0x0000000107b30f31 FRADocumentsMenuController.buildDocumentsMenu() + 4657
1   org.erbele.Erbele             	0x0000000107b313c1 @objc FRADocumentsMenuController.menuNeedsUpdate(_:) + 65
2   com.apple.AppKit              	0x00007fff36602883 -[NSMenu _populateFromDelegateWithEventRef:] + 306
3   com.apple.AppKit              	0x00007fff36519b71 -[NSMenu _populateWithEventRef:] + 81
4   com.apple.AppKit              	0x00007fff3665c028 -[NSCarbonMenuImpl _carbonPopulateEvent:handlerCallRef:] + 360
5   com.apple.AppKit              	0x00007fff3665be51 NSSLMMenuEventHandler + 1197
6   com.apple.HIToolbox           	0x00007fff37cbb8ff DispatchEventToHandlers(EventTargetRec*, OpaqueEventRef*, HandlerCallRec*) + 1254
7   com.apple.HIToolbox           	0x00007fff37cbad8d SendEventToEventTargetInternal(OpaqueEventRef*, OpaqueEventTargetRef*, HandlerCallRec*) + 329
8   com.apple.HIToolbox           	0x00007fff37cbac3d SendEventToEventTargetWithOptions + 45
9   com.apple.HIToolbox           	0x00007fff37d0c2f6 SendMenuPopulate(MenuData*, OpaqueEventTargetRef*, unsigned int, double, unsigned int, OpaqueEventRef*, unsigned char, unsigned char*) + 318
10  com.apple.HIToolbox           	0x00007fff37d0c092 PopulateMenu(MenuData*, OpaqueEventTargetRef*, CheckMenuData*, unsigned int, double) + 100
11  com.apple.HIToolbox           	0x00007fff37d0b73b Check1MenuForKeyEvent(MenuData*, CheckMenuData*) + 226
12  com.apple.HIToolbox           	0x00007fff37d0b3c3 CheckMenusForKeyEvent(MenuData*, CheckMenuData*) + 704
13  com.apple.HIToolbox           	0x00007fff37d0b052 _IsMenuKeyEvent(MenuData*, OpaqueEventRef*, unsigned int, MenuData**, unsigned short*) + 455
14  com.apple.HIToolbox           	0x00007fff37d0ae44 IsMenuKeyEvent + 111
15  com.apple.AppKit              	0x00007fff3681c774 +[NSCarbonMenuImpl _menuItemWithKeyEquivalentMatchingEventRef:inMenu:includingDisabledItems:] + 213
16  com.apple.AppKit              	0x00007fff3665b7ec _NSFindMenuItemMatchingCommandKeyEvent + 163
17  com.apple.AppKit              	0x00007fff36679aa6 -[NSMenu performKeyEquivalent:] + 139
18  com.apple.AppKit              	0x00007fff36aaf9ab routeKeyEquivalent + 521
19  com.apple.AppKit              	0x00007fff364cb87d -[NSApplication(NSEvent) sendEvent:] + 1161
20  org.erbele.Erbele             	0x0000000107abf820 -[FRAApplication sendEvent:] + 4131
21  com.apple.AppKit              	0x00007fff363185bf -[NSApplication run] + 707
22  com.apple.AppKit              	0x00007fff362ea396 NSApplicationMain + 777
23  org.erbele.Erbele             	0x0000000107ab8c33 main + 33
24  libdyld.dylib                 	0x00007fff73151cc9 start + 1
```

**Comment by abentele** [comment]
fixed in commit 06907b51faf8e04e41133a51f4eb8489784a107b and 7553c101907759597d388bdf8c53dbf6442835a4 (pull request pending)

**Comment by abentele** [comment]
Fixed
